processor/k8sattributesprocessor/metadata.yaml

```diff
@@ -375,3 +385,34 @@ telemetry:
       sum:
         value_type: int
         monotonic: true
+    processor_k8sattributes_pod_association_error:
+      enabled: true
+      description: Number of failed pod associations
+      stability:
+        level: development
+      unit: "{resources}"
+      attributes: [k8s_telemetry_type]
+      sum:
+        value_type: int
+        monotonic: true
+    processor_k8sattributes_pod_association_success:
+      enabled: true
+      description: Number of successful pod associations
+      stability:
+        level: development
+      unit: "{resources}"
+      attributes: [k8s_telemetry_type]
+      sum:
+        value_type: int
+        monotonic: true
+    processor_k8sattributes_resource_attributes_added:
+      enabled: true
+      description: Number of times resource attributes were successfully added to resources
```

Comment:
The code looks more like the total number of resource attributes that were added (1).
Can you please explain how we can make sense of this metric? I couldn't find how to get the percentage of data point for which we add attributes or the avg number of resource attributes added by datapoint.

(1) copy of the line of codein the PR (I failed to get the hyperlink of the line of code so I copied it.
```
kp.telemetry.ProcessorK8sattributesResourceAttributesAdded.Add(ctx, int64(finalAttrCount-initialAttrCount), telemetryAttr)
```